### ISSUE TITLE
Report the stack trace when Python exception is thrown from viewer

### ISSUE THREAD

**Issue by yungyuc** [solvcon/solvcon#133]
Some Python exceptions are not properly handled when they are thrown from the viewer, e.g., the stack trace shown in #132:

```
 	KernelBase.dll!00007ff9cb82428c()	Unknown
 	vcruntime140d.dll!00007ff901cab650()	Unknown
>	viewer.exe!pybind11::module_::import(const char * name) Line 1199	C++
 	viewer.exe!modmesh::RApplication::{ctor}::__l2::<lambda_3>::operator()() Line 96	C++
 	[External Code]	
```

I guess it's not the only place to have the issue.

**Comment by tychuang1211** [comment]
Just leaving reference.
[handling-python-exceptions-cpp](https://pybind11.readthedocs.io/en/stable/advanced/exceptions.html#handling-python-exceptions-cpp)

**Comment by tychuang1211** [comment]
I think this issue is solved in #168

**Comment by yungyuc** [comment]
> I think this issue is solved in #168

Thanks.  Let's wait for the feedback from other crashing issues including #115, #132, #169.

**Comment by yungyuc** [comment]
> > I think this issue is solved in #168
> 
> Thanks. Let's wait for the feedback from other crashing issues including #115, #132, #169.

The three issues seem to be more involved than what we solve here.  Close this issue.
